### ISSUE TITLE
[prefer-readonly-type] - Don't report if type is wrapped with ReadOnly utility

### ISSUE THREAD

**Issue by avalanche1** [jonaskello/eslint-plugin-functional#96]
To my knowledge, logically these two are the same:
```ts
type Row = {
 readonly name: string;
 readonly code: string;
}

type Row = Readonly<{
  name: string;
  code: string;
}>
```
The rule reports on the second case. I think it shouldnt.

**Comment by RebeccaStevens** [comment]
Yes, you're correct. We don't current have support for this.

Duplicate of #51
